test/Java/JAR.py

```diff
@@ -134,7 +134,7 @@
 foo.Java(target = 'classes', source = 'com/sub/foo')
 bar.Java(target = 'classes', source = 'com/sub/bar')
 foo.Jar(target = 'foo', source = 'classes/com/sub/foo')
-bar.Jar(target = 'bar', source = 'classes/com/sub/bar')
+bar.Jar(target = 'bar', source = Dir('classes/com/sub/bar'))
```

Comment:
Does this mean if source specify a directory going forward the value will have to be a Dir() instead of a plain string?

Reply:
no this test is testing both ways in this case. We didn't have a test for both cases before, all our tests do plain string.

Reply:
so the Jar above with classes/com/sub/foo is also a directory?

Reply:
Yeah the Java builder will make a directory 'classes' which contains the correct dir structure based off the package for each java file, then compile the java files to class in that classes directory.